google-cloud-bigquery/lib/google/cloud/bigquery/table.rb

```diff
@@ -1217,6 +1239,58 @@ def to_gapi
             @gapi
           end
 
+          ##
+          # Sets the partitioning for the table. See [Partitioned Tables
+          # ](https://cloud.google.com/bigquery/docs/partitioned-tables).
+          #
+          # See {Updater#time_partitioning_type}.
+          #
+          # @param [String] type The partition type. Currently the only
+          # supported value is "DAY".
+          #
+          # @example
+          #   require "google/cloud/bigquery"
+          #
+          #   bigquery = Google::Cloud::Bigquery.new
+          #   dataset = bigquery.dataset "my_dataset"
+          #   table = dataset.create_table "my_table" do |table|
+          #     table.time_partitioning_type = "DAY"
+          #   end
+          #
+          # @!group Schema
```

Comment:
I don't really know what groups are in the docs, but this doesn't seem right as it's not really Schema. Should this be "Attributes" again here as well?